app/routers/tasks.py

```diff
@@ -4,6 +4,10 @@
 import re
 import shutil
 import time
+from app.models.audio_transcription import AudioTranscription
+from app.schemas.audio_transcription import AudioTranscriptionBase
+
+
```

Comment:
@katg999 these also look like unused imports.

Reply:
This was rectified as we need the AudioTranscriptionBase